actor/v7action/security_group_test.go

```diff
@@ -598,7 +599,7 @@ var _ = Describe("Security Group Actions", func() {
 			})
 		})
 
-		When("the request errors", func() {
+		FWhen("the request errors", func() {
```

Comment:
Hey, looks like there's still a focus on this test. That should be removed.

Reply:
I think GitHub Actions can check that for any PR.

Reply:
Or GitHub PR checks

Reply:
Fixed in [1005665](https://github.com/cloudfoundry/cli/pull/2229/commits/b1f6916a865a66b2f7d56e8ce3c7dfd6cc87596e)